docs/guide/components/notes.md

```diff
@@ -10,13 +10,18 @@ Whilst Fractal lets you create rich standalone [documentation pages](../document
 
 ## The README.md file
 
-[Compound components](../components/#compound-components) can include a `README.md` file within their component folder:
+[Components](../components/#components) and [variants](../component/#variants) can include notes in a `README.md`,
+`<component-name>.readme.md`, or `<component-name>--<variant-name>.readme.md` file within their component folder:
```

Comment:
it's a bit messy here. with the new logic, we allow readme files for non-compound components as well, so "within their component folder" is not 100% correct. maybe "beside the component template file" would be better?

for compound components, we allow the main readme file to be added without the component name (similarly to how config files work), so just `readme.md` works as well as `component.readme.md`. this should stay documented as well.